hedera-node/src/main/java/com/hedera/services/store/TopicStore.java

```diff
@@ -62,4 +67,38 @@ public void persistNew(Topic model) {
 		currentTopics.put(id, merkleTopic);
 		transactionRecordService.includeChangesToTopic(model);
 	}
+
+	/**
+	 * Performs loading of a given merkle topic from state. The merkle topic is mapped to a model.
+	 * Please note that the model should be explicitly persisted with {@link TopicStore#persistTopic}
+	 * in order to apply the changes made.
+	 *
+	 * @param id
+	 * 		- model ID of the topic to be loaded.
+	 * @return {@link Topic} - the loaded topic
+	 */
+	public Topic loadTopic(Id id) {
+		final var entityNum = id.asEntityNum();
+		final var currentTopics = topics.get();
+		final var merkleTopic = currentTopics.get(entityNum);
+
+		validateUsable(merkleTopic);
+		return fromMerkle(merkleTopic, id);
+	}
+
+	/**
+	 * <p>Persists an updated {@link Topic} into the state, as well as exporting its {@link Id} to the transaction receipt.</p>
+	 * @param source
+	 * 		<p>the model to be mapped onto an updated {@link MerkleTopic} and persisted.</p>
+	 */
```

Comment:
Nit: let's be consistent about the documentation formatting - either <p> tags or no tags at all